modules/Bio/EnsEMBL/Analysis/Hive/Config/GenblastHomology.pm

```diff
@@ -0,0 +1,532 @@
+                         top_level => 1,
+                         min_slice_length => $self->o('min_toplevel_slice_length'),
+                         batch_slice_ids => 1,
+                         batch_target_size => 20000000,
+                       },
+        -rc_name    => '2GB',
+        -flow_into => {
+                         '2'    => ['remove_redundant_genblast_genes'],
+                      },
+      },
+
+
+     {
+      -logic_name => 'remove_redundant_genblast_genes',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::RemoveRedundantGenes',
+      -parameters => {
+        target_db => $self->o('genblast_nr_db'),
+        target_type => 'biotype_priority',
+        layers => get_analysis_settings('Bio::EnsEMBL::Analysis::Hive::Config::LayerAnnotationStatic', $self->o('uniprot_set'), undef, 'ARRAY'),
+      },
+      -rc_name          => '5GB',
+     },
+
+    ];
+}
+
+
+sub resource_classes {
+  my $self = shift;
+
+  return {
+    '2GB' => { LSF => $self->lsf_resource_builder('production-rh74', 2000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '3GB' => { LSF => $self->lsf_resource_builder('production-rh74', 3000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '4GB' => { LSF => $self->lsf_resource_builder('production-rh74', 4000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '5GB' => { LSF => $self->lsf_resource_builder('production-rh74', 5000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'default' => { LSF => $self->lsf_resource_builder('production-rh74', 900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'genblast' => { LSF => $self->lsf_resource_builder('production-rh74', 3900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'genblast_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'genblast_retry' => { LSF => $self->lsf_resource_builder('production-rh74', 4900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'genblast_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+  }
+}
+
+sub hive_capacity_classes {
+  my $self = shift;
+
+  return {
+           'hc_very_low'    => 35,
+           'hc_low'    => 200,
+           'hc_medium' => 500,
+           'hc_high'   => 1000,
+         };
+}
+
+
+sub check_file_in_ensembl {
+  my ($self, $file_path) = @_;
+  push @{$self->{'_ensembl_file_paths'}}, $file_path;
+  return $self->o('enscode_root_dir').'/'.$file_path;
+}
```

Comment:
This can be removed as it is not used anywhere

Reply:
Method removed